tests/cecil-tests/AttributeTest.cs

```diff
@@ -163,36 +473,31 @@ public void AllAttributedItemsMustIncludeCurrentPlatform (string assemblyPath)
 				return;
 			}
 
-#if DEBUG
-			Console.WriteLine(assemblyPath);
-#endif
+			if (!assemblyPath.Contains("Xamarin.Mac.dll")) {
```

Comment:
There's no "Xamarin.Mac.dll" anymore for .NET (it's "Microsoft.macOS.dll"), so this seems wrong.

Reply:
This was a cludge to only run one test more quickly. 🤦 

Let me remove and re-run

Reply:
Fixed!